src/utilities/src/util.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 The Manifold Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include <thrust/iterator/counting_iterator.h>
+
+namespace manifold {
+thrust::counting_iterator<int> countAt(int i) {
+  return thrust::make_counting_iterator(i);
```

Comment:
Do we really need a whole extra file for this tiny function? Can't we just inline it in the header?

Reply:
True, I will change it into an inline function